ddtrace/_monkey.py

```diff
@@ -146,6 +147,7 @@
     "httplib": ("http.client",),
     "kafka": ("confluent_kafka",),
     "google_generativeai": ("google.generativeai",),
+    "langgraph": ("langgraph.graph",),
```

Comment:
Will this not be patched if someone imports `langgraph.pregel/func/channels/prebuilt` or other top-level langgraph directories?

Reply:
ah hmm i suppose so, i forgot i had this here. without this, the patching was never triggered - i only added it in after trying to figure out, without too much understanding, how exactly patching is triggered in python. so, i'm not sure what i'd change this to here

i think i might defer to you as i don't have too much knowledge on how the patching hooks are triggered 😅 but we can run thru it offline too, i'd love to learn how it works.

Reply:
Yeah let's talk about this offline once you're back, happy to walk through patching hooks and underlying instrumentation logic in Python 😄